frontend/components/site/sitePublishedBranchesTable.js

```diff
@@ -62,7 +64,7 @@ class SitePublishedBranchesTable extends React.Component {
   }
 
   renderLoadingState() {
-    return <p>No published branch data available</p>
+    return <LoadingIndicator/>
```

Comment:
Okay wait, I put this in the wrong spot. I thought this table had an empty state 😳